static/sass/_pattern_shop-cart.scss

```diff
@@ -12,11 +12,19 @@
       display: flex;
       flex-wrap: wrap;
 
+      > .p-label {
+        margin-top: 0.5rem;
+        align-self: center;
```

Comment:
Linter is failing. I think it's because properties are not in alphabetical order.